packages/js/product-editor/src/blocks/product-fields/custom-fields-toggle/block.json

```diff
@@ -0,0 +1,27 @@
+{
+	"$schema": "https://schemas.wp.org/trunk/block.json",
+	"apiVersion": 2,
+	"name": "woocommerce/product-custom-fields-toggle-field",
+	"title": "Product custom fields toggle control",
+	"category": "woocommerce",
+	"description": "The product custom fields toggle.",
+	"keywords": [ "products", "custom", "fields" ],
+	"textdomain": "default",
+	"attributes": {
+		"label": {
+			"type": "string",
+			"__experimentalRole": "content"
+		}
+	},
+	"supports": {
+		"align": false,
+		"html": false,
+		"multiple": true,
+		"reusable": true,
+		"inserter": false,
+		"lock": false,
+		"__experimentalToolbar": false
+	},
+	"usesContext": [ "postType" ],
```

Comment:
I think that we can remove this line. We won't use the `context.postType` in the block.

Reply:
done -> https://github.com/woocommerce/woocommerce/pull/45360/commits/869834b96df3a9c9baeb13142e345998185eef07